dataactcore/scripts/migrateDataBroker.py

```diff
@@ -0,0 +1,68 @@
+# migrate data using pg_dump and pg_restore
+# data copied from tables:
+# error_data:
+#   error_metadata
+#   file
+# job_tracker:
+#   job
+#   submission
+#   job_dependency
+# user_manager;
+#   users
+#   email_token
+# validator:
+#   appropriation
+#   award_financial
+#   award_financial_assistance
+#   object_class_program_activity
+
+# run on command line: python migrateDataBroker.py
+
+from dataactcore.config import CONFIG_DB
+import subprocess
+
+c = 'postgresql://{}:{}@{}/'.format(
+    CONFIG_DB['username'], CONFIG_DB['password'], CONFIG_DB['host'])
+target = '{}data_broker'.format(c)
+
+# error_data
+db = 'error_data'
+source = '{}{}'.format(c, db)
+print('migrating {}'.format(db))
+cmd = 'pg_dump -d {} -t error_metadata -t file --data-only --format=c | ' \
+    'pg_restore -d {} --data-only'.format(source, target)
+p = subprocess.call(cmd, shell=True, stdout=subprocess.PIPE)
+print('return code = {}\n'.format(p))
+
```

Comment:
We would also need the 'file' table here for check_status to continue to work normally.  This table holds information about whether a file completed validation or stopped with header errors, etc.